### PR TITLE
Added a Jira action for adding multiple attachments to an issue

### PR DESCRIPTION
**Pull request by justabuzz**
This PR contains:
1. New Jira action for adding multiple attachments to an existing issue
2. A small typo fix in the description of the Jira action `add-attachment-to-issue`

<a href="https://gitpod.io/#https://github.com/PipedreamHQ/pipedream/pull/4041"></a>

### CODE REVIEW

**Comment by dylburger** [comment]
Thank you so much for submitting this! We've added it to our backlog to review, and our team has been notified.

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible

**Review by vellames — DISMISSED** [review]

**Comment by vunguyenhung** [comment]
Hello everyone, I have tested this PR and there're some test cases failed or needed improvement. 

Please check the test report below for more information
[JIRA_4041_2215.pdf](https://github.com/PipedreamHQ/pipedream/files/9383243/JIRA_4041_2215.pdf)

File: components/jira/actions/add-multiple-attachments-to-issue/add-multiple-attachments-to-issue.mjs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  },
+  async run({ $ }) {
+    const responses = [];
+
+    //validate all files exist before processing
+    const not_found = [];
+    for (let i = 0; i < this.filenames.length; i++) {
+      const path = utils.checkTmp(this.filenames[i]);
+      if (!fs.existsSync(path)) {
+        not_found.push(path)
```
Comment (feyzullah) [inline components/jira/actions/add-multiple-attachments-to-issue/add-multiple-attachments-to-issue.mjs:35]:
```suggestion
        not_found.push(path);
```

File: components/jira/actions/add-multiple-attachments-to-issue/add-multiple-attachments-to-issue.mjs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      }
+    }
+
+    if (not_found.length > 0) {
+      throw new ConfigurationError(`File${not_found.length > 1 ? "s" : ""} not found: ${not_found.join(", ")}`);
```
Comment (feyzullah) [inline components/jira/actions/add-multiple-attachments-to-issue/add-multiple-attachments-to-issue.mjs:40]:
```suggestion
      // eslint-disable-next-line multiline-ternary
      throw new ConfigurationError(`File${not_found.length > 1 ? "s" : ""} not found: ${not_found.join(", ")}`);
```

**Review by feyzullah — APPROVED** [review]
LGTM! Just added linter related suggestions

**Review by feyzullah — APPROVED** [review]

**Comment by vunguyenhung** [comment]
Hi everyone, all test cases are passed! Ready for release! 

Test report
[JIRA_4041_2042.pdf](https://github.com/PipedreamHQ/pipedream/files/9402987/JIRA_4041_2042.pdf)

**Pull request merged**